AutoRouteManager.php

```diff
@@ -50,11 +55,13 @@ class AutoRouteManager
     public function __construct(
         AdapterInterface $adapter,
         UriGeneratorInterface $uriGenerator,
-        DefunctRouteHandlerInterface $defunctRouteHandler
+        DefunctRouteHandlerInterface $defunctRouteHandler,
+        UriContextCollectionBuilder $collectionBuilder
```

Comment:
There is already a BC break in `master`, I wonder if we should just stick to semver and release 2.0 next.